rotkehlchen/api/v1/schemas.py

```diff
@@ -804,6 +804,7 @@ class ModifiableSettingsSchema(Schema):
         validate=lambda data: len(data) == len(set(data)),
     )
     cost_basis_method = SerializableEnumField(enum_class=CostBasisMethod, load_default=None)
+    eth2_taxable_after_withdrawal_enabled = fields.Bool(load_default=None)
```

Comment:
```suggestion
    eth2_taxable_after_withdrawal_enabled = fields.Boolean(load_default=None)
```

It is actually the same but we use `Boolean` in our codebase

Reply:
👍